src/Microsoft.DocAsCode.Dfm/DfmRenderer.cs

```diff
@@ -131,6 +131,30 @@ public virtual StringBuffer Render(IMarkdownRenderer renderer, DfmVideoBlockSpli
             StringBuffer content = string.Empty;
 
             var videoToken = splitToken.Token as DfmVideoBlockToken;
+            //TODO: fix video link
+            //if youtube.com, change to youtube-nocookie.com
+            //if channel9.msdn.com, add ?nocookie=true at end
+            if (videoToken != null && !String.IsNullOrWhiteSpace(videoToken.Link))
```

Comment:
>String [](start = 39, length = 6)

use string?

Reply:
not sure what you are suggesting? string vs. String ? Given that I'm using a static method of the framework class this seems appropriate vs. string, but I can change if it matches your desired coding guidelines